inspirehep/modules/theme/templates/inspirehep_theme/conferences_in_series.html

```diff
@@ -0,0 +1,27 @@
+{# 
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+#}
+
+<div id="other-conferences-in-the-series">
+  <a href="/conferences/{{record.control_number}}"> {{ record.title }}</a>
+</div>
```

Comment:
I think rather than using yet another templating language we should try and
now go full angular for such use cases in an effort to reduce the number of
dependencies.

On Thu, 26 May 2016 09:13 Jacopo Notarstefano, <notifications@github.com>
wrote:

> In
> inspirehep/modules/theme/templates/inspirehep_theme/conferences_in_series.html
> <https://github.com/inspirehep/inspire-next/pull/1138#discussion_r64700354>
> :
>
> > +# INSPIRE is distributed in the hope that it will be useful,
> > +# but WITHOUT ANY WARRANTY; without even the implied warranty of
> > +# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
> > +# GNU General Public License for more details.
> > +#
> > +# You should have received a copy of the GNU General Public License
> > +# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
> > +#
> > +# In applying this licence, CERN does not waive the privileges and immunities
> > +# granted to it by virtue of its status as an Intergovernmental Organization
> > +# or submit itself to any jurisdiction.
> > +#}
> > +
> > +<div id="other-conferences-in-the-series">
> > +  <a href="/conferences/{{record.control_number}}"> {{ record.title }}</a>
> > +</div>
>
> These templates are simple, so they could also be done using mustache,
> which would spare us the weird thing of rendering stuff server-side and
> then sending the HTML back to the client... what is the reasoning for doing
> it like that?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/inspirehep/inspire-next/pull/1138/files/fd48f0f380dcce8aec3c100dfc0c02acf9ff28d5#r64700354>
>